android/keymanager/src/main/java/io/mosip/registration/keymanager/util/CryptoUtil.java

```diff
@@ -45,4 +45,15 @@ public static String generateMD5Hash(byte[] data) {
         return new String(Hex.encodeHex(messageDigestMD5.digest(data)));
     }
 
+    public static byte[] decodeBase64(String data) {
+        if (data == null || data.trim().length() == 0) {
```

Comment:
we should use this https://commons.apache.org/proper/commons-lang/apidocs/org/apache/commons/lang3/StringUtils.html#isBlank-java.lang.CharSequence-m since we already have this library as part of our dependencies

Reply:
Used suggested method.